SU-SWS/stanford_profile
# D8CORE-3520 All Publications list page

# READY FOR REVIEW

# Summary
- created the /publications list page
- I had to put the default content here instead of the publications module because it relies on the paragraph type from the profile. So it doesn't work to put it into the module.

# Related PRS
https://github.com/SU-SWS/stanford_publication/pull/28
https://github.com/SU-SWS/stanford_profile_helper/pull/74


# Need Review By (Date)
- 3/3

# Urgency
- high

# Steps to Test
1. `composer require su-sws/stanford_profile:dev-D8CORE-3520 --no-update`
2. `composer update -n`
3. either `blt di -n` or `blt drupal:update` depending if you have an existing site
4. verify you have a `/publications` page created.
5. create some publication topic terms and some publication content. (devel generate works good)
6. ensure they show up on the /publications page.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
